test/e2e/infinispan/smoke_test.go

```diff
@@ -52,6 +53,7 @@ func TestBaseFunctionality(t *testing.T) {
 				WhenUnsatisfiable: "ScheduleAnyway",
 			}},
 		}
+		i.Spec.Service.Container.TerminationGracePeriodSeconds = pointer.Int64(60)
```

Comment:
@pminz I have pushed a commit to simplify this code. The `pointer` package is very nice for these things as it means you can do assignment with one line instead of two.